tasks/check-type-imports.js

```diff
@@ -0,0 +1,96 @@
+import fs from 'fs';
+import path from 'path';
+import chalk from 'chalk';
+import monoglob from 'glob';
+import ts from 'typescript';
+
+import { getConstructorNode, getTypesFromConstructor } from '../cem/support-typedef-jsdoc-utils.js';
+
+function glob (patternList, options) {
+  if (typeof patternList === 'string') {
+    return monoglob.sync(patternList, options);
+  }
+  return patternList.flatMap((pattern) => {
+    return monoglob.sync(pattern, options);
+  });
+}
```

Comment:
It seems like you're not using the muti version anymore, can we simplify this?

Reply:
I don't understand what you mean by multi ? :thinking: